### DISCUSSION TITLE
Community support for TEP-0042

### DISCUSSION THREAD

**Discussion by bigfleet** [tektoncd/community#468] [General]
I've just watched a presentation at cdCon from @vdemeester (and a colleague, I was too slow to capture his @ sign) showing progress on the PoC for this TEP.   I have been writing Jenkins builds for many years, and this functionality was very exciting to see.  The roadmap seems very vibrant.  I don't presume to comment on the quality of the implementation-- I simply do not have the capability to judge-- but this capability would be very exciting to expose to the many engineers who will, in time, produce working tekton pipelines.  

Vincent mentioned during the Q&A that this was the second time he had received this feedback, so I wanted to both provide a kudos to the team for making it as far as this, and to provide a landing place for future 👍🏼 activity from those without commit bits.

Cheers!

**Comment by vdemeester** [comment]
cc @bobcatfish @afrittoli Indeed, I wonder if we should have a discussion opened per TEP that is in proposed state to have a way discuss feedback on it ?
